src/Subscriptions/ViewModels/SubscriptionViewModel.php

```diff
@@ -19,18 +24,76 @@ public function __construct(Subscription $subscription)
         $this->subscription = $subscription;
     }
 
+    /**
+     * @unreleased
+     */
+    public function includeSensitiveData(bool $includeSensitiveData = true): SubscriptionViewModel
+    {
+        $this->includeSensitiveData = $includeSensitiveData;
+
+        return $this;
+    }    
+
+    /**
+    * @unreleased
+    */
+    public function anonymousMode(DonorAnonymousMode $mode): SubscriptionViewModel
+    {
+        $this->anonymousMode = $mode;
+
+        return $this;
+    }
+
     /**
      * @unreleased
      */
     public function exports(): array
     {
+        $donor = $this->subscription->donor()->get();
```

Comment:
If a model has a defined relationship, then you can just call the property and it will return the model for you.  Alternatively, invoking the method will return the query builder.

```suggestion
        $donor = $this->subscription->donor;
```

Reply:
Resolved in 2fd4d66ec9d50b26537674276ec0c56cbc18169b